test/views/declaration/spec/UnitViewSpec.scala

```diff
@@ -43,6 +47,31 @@ class UnitViewSpec extends UnitSpec with ViewMatchers {
   }
 
   def messagesKey(key: String): BeMatcher[String] = new MessagesKeyMatcher(key)
+
+  def onEveryDeclarationJourney(f: JourneyRequest[_] => Unit): Unit =
+    onJourney(DeclarationType.STANDARD, DeclarationType.SUPPLEMENTARY, DeclarationType.SIMPLIFIED)(f)
+
+  def onJourney(types: DeclarationType*)(f: JourneyRequest[_] => Unit): Unit =
```

Comment:
I get it now.  This would have been good to have a while ago.   I'm wondering how many of the existing tests I need to replicate for the new OCCASIONAL dec type.